_data/news.yml

```diff
@@ -1,3 +1,7 @@
+- date: 31 December 2023
+  headline: "Google Season of Docs 2023 Concluded Successfully"
+  link: "/blogs/gsod23_quillpusher_experience_blog/"
```

Comment:
I don't understand the link? Why the trailing / and not something like .html?

Reply:
That's how relative links work in the Front Matter, as I understand
I just checked, previous blog entries and other pages follow the same format (except for the allnew.html page, where it uses the explicit link instead)